src/ui/server/controllers/insurance/export-contract/map-submitted-data/export-contract/index.test.ts

```diff
@@ -94,4 +97,52 @@ describe('controllers/insurance/export-contract/map-submitted-data/export-contra
       expect(result).toEqual(expected);
     });
   });
+
+  describe(`when ${AWARD_METHOD} is not equal to ${OTHER_DB_ID}`, () => {
+    it(`should return the form body with an empty ${OTHER_AWARD_METHOD}`, () => {
+      const mockBody = {
+        ...mockFormBody,
+        [AWARD_METHOD]: mockAwardMethodId,
+        [OTHER_AWARD_METHOD]: 'test',
```

Comment:
Could we move this into UI test mocks in `mockExportContract` with a value of "Mock other award method"